### ISSUE TITLE
Error during polling

### ISSUE THREAD

**Issue by crxporter** [NRCHKB/node-red-contrib-unifi-os#25]
Shaquu  - as promised, here's the error I'm getting about once per hour during polling. No interaction is required, the node re-authenticates and continues running happily. I'm polling every 2 seconds.

<img width="383" alt="image" src="https://user-images.githubusercontent.com/38265886/125874856-4a8f0bf9-e527-4f6b-8da6-7134d9c41ec7.png">

**Comment by pshellguruwannabe** [comment]
I"m getting this same error 401.  have to update credentials and restart flow to get it working again.  seems like it stops working after 2 or 3 cycles.

**Comment by pshellguruwannabe** [comment]
> I"m getting this same error 401. have to update credentials and restart flow to get it working again. seems like it stops working after 2 or 3 cycles.

I figured out part of my problem.  I uninstalled the old Unifi modules (for USG).  Like @crxporter i'm getting authentication errors once an hour and it resumes without intervention.  issue i have is i have some flows that only run once a day and if it hits during that time, it doesn't run and i have to manually intervene.

**Comment by crxporter** [comment]
Update about the hourly error: @Shaquu has a new plan to make a periodic login update every 45 minutes. This should be ready soon I think? 

We hope it will take care of the hourly 401

**Comment by pshellguruwannabe** [comment]
I updated to the latest version a couple days ago.  I'm still getting the error every hour still.  Was the latest update supposed to fix this issue, or something else maybe?

**Comment by crxporter** [comment]
No this wasn't included in 0.6.0 ... still waiting on this one. I'll see if I can encourage  Mr Shaquu to have a look

**Comment by pshellguruwannabe** [comment]
No worries.  Just checking.

**Comment by Shaquu** [comment]
Please check 0.7.0-dev.4

**Comment by pshellguruwannabe** [comment]
don't see dev4.  should i grab dev5?

**Comment by pshellguruwannabe** [comment]
So, i did find dev4, installed it.  still getting error about once an hour.  installed dev5, same thing.  Also, somewhere along the way, updating wlanconf has broken.  i have a flow that updates the guest network password and it now returns and error and does not update the password.  This is what i get back:

Bad response from: https://192.168.1.254/proxy/network/api/s/default/rest/wlanconf/xxxxxxxxxxxxxxxxxxxxxxx

Error: Invalid Payload undefined undefined

Nothing has changed on that flow.  If i need to open a new issue for that, just let me know.

**Comment by Shaquu** [comment]
@pshellguruwannabe this new error might be related to UniFi update itself

**Comment by Shaquu** [comment]
@pshellguruwannabe please try 0.7.0-dev.6 :)
If not, we will iterate again.

I am testing myself now.

**Comment by pshellguruwannabe** [comment]
the error has stopped appearing hourly for login.
